src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/paymentFromComponent.js

```diff
@@ -144,6 +144,17 @@ function handleExpressPayment(reqDataObj, currentBasket) {
   }
 }
 
+function canSkipSummaryPage(reqDataObj) {
+  if (
+    AdyenHelper.isApplePay(reqDataObj.paymentMethod?.type) ||
```

Comment:
Instead of checking per payment method, it would be better to have one array and check whether this payment method type is in there.